dateutil/test/test_parser.py

```diff
@@ -303,11 +308,10 @@ def testDateCommandFormatReversed(self):
                                   tzinfo=self.brsttz))
 
     def testDateCommandFormatWithLong(self):
```

Comment:
I think maybe just add `@pytest.mark.skipif(six.PY3, reason="Long does not exist in Python 3")`

This test is specifically for support of Python 2 longs.